Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -392,6 +392,16 @@ - (BOOL)textInputShouldChangeTextInRange:(NSRange)range replacementText:(NSStrin
                                    notifyDelegate:YES];
 
         [self textInputDidChange];
+        
+        _nativeEventCount++;
+        
+        if (_onChange) {
```

Comment:
I am not comfortable with calling `onChange` inside `textInputShouldChangeTextInRange`.

As I said previously, all quicky behavior should be implemented in delegate adapter classes.
I do want to fix this issue very very badly but we cannot sacrifice maintainability of the code for this.